src/odemis/gui/test/comp_canvas_test.py

```diff
@@ -600,11 +600,11 @@ def test_pyramidal_zoom(self):
 
         result_im = get_image_from_buffer(self.canvas)
         px2 = get_rgb(result_im, result_im.Width // 2, result_im.Height // 2)
-        # center pixel, 2/3 green, 1/3 blue. The green image is the largest image
-        self.assertEqual(px2, (0, 179, 76))
+        # center pixel, half green, half blue. The green image is the largest image
+        self.assertEqual(px2, (0, 127, 128))
         px2 = get_rgb(result_im, result_im.Width // 2 - 30, result_im.Height // 2 - 30)
-        # background of the images, 2/3 green, 1/3 red
-        self.assertEqual(px2, (76, 179, 0))
+        # background of the images, half green, half red
+        self.assertEqual(px2, (128, 127, 0))
```

Comment:
I don't get why the order of the color of the pixel (0, 127, 128) is different here than in the test case above.

Reply:
I can add some comments. `get_rgb` takes as input arguments `image, x, y`, it then returns the value of that pixel. The image is half green, half red and has a blue-green square in the middle. This `result_im.Width // 2 - 30, result_im.Height // 2 - 30` just gets the rgb value of a pixel 30 pixels to the top left of the center.